docs/xcompilationthreads.md

```diff
@@ -31,7 +31,7 @@ Use this option to specify the number of compilation threads that are used by th
 
         -XcompilationThreads<n>
 
-: where `<n>` is the number of threads, in the range 1-4 inclusive. A number greater than 4 prevents the Java&trade; VM from starting successfully. 
+: where `<n>` is the number of threads, in the range 1-7 inclusive. Any number outside this range will be ignored. 
```

Comment:
Please can you change the "will be" to "is" for consistency with the rest of the docs. Future tense should be avoided, where possible.

Reply:
Done. Ready for another review.